xsd/netex_part_1/part1_ifopt/netex_ifopt_site_version.xsd

```diff
@@ -567,6 +567,11 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Short Name of LEVEL.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="OrderNumber" type="xsd:integer" minOccurs="0">
```

Comment:
usually he name is only "order" in NeTEx

Reply:
...and it is an attribute